<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SHA-LRI-PC-264" name="Default test(failed)" tests="1" failures="1" timestamp="14 Aug 2018 09:47:28 GMT" time="0.025" errors="0">
  <testcase name="testRandomEmail2" time="0.013" classname="trainTestNG.EmailGeneratorTest">
    <failure type="java.lang.AssertionError" message="expected [rndom@trainTestNG.com] but found [random@trainTestNG.com]">
      <![CDATA[java.lang.AssertionError: expected [rndom@trainTestNG.com] but found [random@trainTestNG.com]
	at trainTestNG.EmailGeneratorTest.testRandomEmail2(EmailGeneratorTest.java:21)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testRandomEmail2 -->
</testsuite> <!-- Default test(failed) -->
